seedlet/functions.php

```diff
@@ -396,7 +386,9 @@ function seedlet_skip_link_focus_fix() {
 /**
  * Custom colors class.
  */
-require get_template_directory() . '/classes/class-seedlet-custom-colors.php';
+if ( empty( get_theme_mod( 'colors_manager' ) ) ) { // If the theme is on wpcom, we bypass the theme's built in custom colors, because wpcom uses a different custom color implementation.
```

Comment:
Is this an acceptable way to "unload" the theme's default custom color implementation?

Reply:
Yeah, I think it's okay.